azkaban-common/src/main/java/azkaban/executor/ExecutionLogsCleaner.java

```diff
@@ -0,0 +1,69 @@
+package azkaban.executor;
```

Comment:
My initial thought process was to move the code out and use it in ExecutorManager and ExecutionController. That is why I had placed it here. Now that it runs independently we needn't keep it here. Moved it to web server module. 

My earlier comments were lost because I typed comments and resolve in github thinking that those comments will be recorded!